### PR TITLE
Option for emitting enum aliases.

### PR DESCRIPTION
**Pull request by Manuzor**
These changes add an option to emit aliases for enums to "export" them to module-scope. Like this:

``` d
enum VkResult
{
  VK_SUCCESS = 0,
  // ...
}
alias VK_SUCCESS = VkResult.VK_SUCCESS; // This is optional
// ...
```

I know this makes the output more verbose but I'd really like to have this option, so I implemented it. It makes for more similarity with C/C++ Vulkan programs. Let me know what you think about it.

### CODE REVIEW

**Comment by ColonelThirtyTwo** [comment]
Please fix your commit so that it does not convert newlines (and thus marks the entire file as changed).

**Comment by Manuzor** [comment]
Fixed, sorry about that. At some point, I seem to have converted the line endings to `LF` when they originally were `CRLF`.

File: vkdgen.py [hunk 1]
```diff
@@ -256,8 +256,9 @@ def genStruct(self, typeinfo, name):
 		print("}", file=self.typesFile)
 	
 	def genGroup(self, groupinfo, name):
-		super().genGroup(groupinfo, name)
-		print("enum %s {" % name, file=self.typesFile)
+		enumName = name
```
Comment (Manuzor) [inline vkdgen.py:259]: In case you're wondering, I need to save the original enum name here so that I can use it in line 306.

**Comment by ColonelThirtyTwo** [comment]
Added in 00c866a4d0b40982cad83287eb3440b0a1d3b8a7

**Pull request closed without merging**